packages/@react-aria/interactions/test/useHover.test.js

```diff
@@ -316,7 +316,9 @@ describe('useHover', function () {
         {
           type: 'hoverstart',
           target: el,
-          pointerType: 'mouse'
+          pointerType: 'mouse',
```

Comment:
I've been mulling over the use of x,y for hover. I don't think it makes much sense because it'll only be a coordinate for the start and end, none of the movement between the two.

Do you have a specific need for it? Maybe we can leave it out of the PR until someone requests it specifically